src/Uno.UI.Tests/Windows_Globalization/Given_NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,1622 @@
+		[DataRow("ar-SA", "Arab", "ar-SA")]
+		[DataRow("ar-SO", "Arab", "ar-SO")]
+		[DataRow("ar-SS", "Arab", "ar-SS")]
+		[DataRow("ar-SD", "Arab", "ar-SD")]
+		[DataRow("ar-SY", "Arab", "ar-SY")]
+		[DataRow("ar-TN", "Latn", "ar-TN")]
+		[DataRow("ar-AE", "Arab", "ar-AE")]
+		[DataRow("ar-001", "Arab", "ar-001")]
+		[DataRow("ar-YE", "Arab", "ar-YE")]
+		[DataRow("hy", "Latn", "hy")]
+		[DataRow("hy-AM", "Latn", "hy")]
+		[DataRow("as", "Latn", "as")]
+		[DataRow("as-IN", "Latn", "as")]
+		[DataRow("ast", "Latn", "ast")]
+		[DataRow("ast-ES", "Latn", "ast-Latn-ES")]
+		[DataRow("asa", "Latn", "asa")]
+		[DataRow("asa-TZ", "Latn", "asa-Latn-TZ")]
+		[DataRow("az-Cyrl", "Latn", "az-Cyrl")]
+		[DataRow("az-Cyrl-AZ", "Latn", "az-Cyrl")]
+		[DataRow("az", "Latn", "az")]
+		[DataRow("az-Latn", "Latn", "az-Latn")]
+		[DataRow("az-Latn-AZ", "Latn", "az-Latn")]
+		[DataRow("ksf", "Latn", "ksf")]
+		[DataRow("ksf-CM", "Latn", "ksf-Latn-CM")]
+		[DataRow("bm", "Latn", "bm")]
+		[DataRow("bm-Latn-ML", "Latn", "bm-Latn-ML")]
+		[DataRow("bn", "Beng", "bn")]
+		[DataRow("bn-BD", "Beng", "bn-BD")]
+		[DataRow("bn-IN", "Latn", "bn-IN")]
+		[DataRow("bas", "Latn", "bas")]
+		[DataRow("bas-CM", "Latn", "bas-Latn-CM")]
+		[DataRow("ba", "Latn", "ba")]
+		[DataRow("ba-RU", "Latn", "ba-Cyrl")]
+		[DataRow("eu", "Latn", "eu")]
+		[DataRow("eu-ES", "Latn", "eu")]
+		[DataRow("be", "Latn", "be")]
+		[DataRow("be-BY", "Latn", "be")]
+		[DataRow("bem", "Latn", "bem")]
+		[DataRow("bem-ZM", "Latn", "bem-Latn-ZM")]
+		[DataRow("bez", "Latn", "bez")]
+		[DataRow("bez-TZ", "Latn", "bez-Latn-TZ")]
+		[DataRow("byn", "Latn", "byn")]
+		[DataRow("byn-ER", "Latn", "byn-Ethi-ER")]
+		[DataRow("brx", "Latn", "brx")]
+		[DataRow("brx-IN", "Latn", "brx-Deva-IN")]
+		[DataRow("bs-Cyrl", "Latn", "bs-Cyrl")]
+		[DataRow("bs-Cyrl-BA", "Latn", "bs-Cyrl")]
+		[DataRow("bs-Latn", "Latn", "bs")]
+		[DataRow("bs", "Latn", "bs")]
+		[DataRow("bs-Latn-BA", "Latn", "bs")]
+		[DataRow("br", "Latn", "br")]
+		[DataRow("br-FR", "Latn", "br-Latn")]
+		[DataRow("bg", "Latn", "bg")]
+		[DataRow("bg-BG", "Latn", "bg")]
+		[DataRow("my", "Mymr", "my")]
+		[DataRow("my-MM", "Mymr", "my")]
+		[DataRow("ca", "Latn", "ca")]
+		[DataRow("ca-AD", "Latn", "ca-AD")]
+		[DataRow("ca-FR", "Latn", "ca-FR")]
+		[DataRow("ca-IT", "Latn", "ca-IT")]
+		[DataRow("ca-ES", "Latn", "ca")]
+#if !NETFX_CORE
+		[DataRow("ceb", "ArabExt", "fa")]
+		[DataRow("ceb-Latn", "ArabExt", "fa")]
+		[DataRow("ceb-Latn-PH", "ArabExt", "fa")]
+#endif
+		[DataRow("tzm-Arab-MA", "Latn", "tzm-Arab-MA")]
+		[DataRow("tzm-Latn-MA", "Latn", "tzm-Latn-MA")]
+		[DataRow("ku", "Arab", "ku")]
+		[DataRow("ku-Arab", "Arab", "ku-Arab")]
+		[DataRow("ku-Arab-IQ", "Arab", "ku-Arab")]
+#if !NETFX_CORE
+		[DataRow("ccp", "ArabExt", "fa")]
+		[DataRow("ccp-Cakm", "ArabExt", "fa")]
+		[DataRow("ccp-Cakm-BD", "ArabExt", "fa")]
+		[DataRow("ccp-Cakm-IN", "ArabExt", "fa")]
+#endif
+		[DataRow("ce-RU", "Latn", "ce-Cyrl-RU")]
```

Comment:
@jeromelaban I disabled tests for UWP. Did I do this correctly?

Reply:
@jeromelaban  On my computer number of failed tests was 9. I don't know why it was 11 for you. Please tell me if tests failed again.